test/distributed/fsdp/test_fsdp_grad_acc.py

```diff
@@ -132,8 +162,13 @@ def permute_tensor(x: torch.Tensor):
 
     @skip_if_lt_x_gpu(2)
     @parametrize(
-        "num_iters_to_acc",
-        [2, 4],
+        "configs",
+        [
+            [_GradAccConfig(True, 4)],
+            [_GradAccConfig(False, 4)],
+            [_GradAccConfig(True, 2), _GradAccConfig(False, 2), _GradAccConfig(True, 2)],
+            [_GradAccConfig(False, 2), _GradAccConfig(True, 2), _GradAccConfig(False, 2)],
```

Comment:
Wondering why we have duplicated configs?

Reply:
I wanted to test interleaving both ways:
1) with `no_sync()` -> without `no_sync()` -> with `no_sync()`
2) without `no_sync()` -> with `no_sync()` -> without `no_sync()`

The reason I wanted two separate tests is that it _could_ matter what was the last accumulation mode right before gradient synchronization.

Reply:
Good call! Let's keep it then and add a small comment so no one removes it in the future.